RELEASING.md

```diff
@@ -0,0 +1,77 @@
+## Release and Branch Management
```

Comment:
documents must start with a `#` header level, what do you think about reducing all headings by 1?